### PR TITLE
discord login api

### PR DESCRIPTION
**Pull request by YashVardhan-AI**

### CODE REVIEW

**Comment by yashppawar** [comment]
Any updates @porus-creator ?

**Review by yashppawar — APPROVED** [review]

File: static/scripts/script.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// function to ADD two numbers
```
Comment (shravanasati) [inline static/scripts/script.js:1]: huh!?

File: encrypt.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    encryptedDict = {
+        'cipher_text': b64encode(cipher_text).decode('utf-8'),
+        'salt': b64encode(salt).decode('utf-8'),
+        'nonce': b64encode(cipher_config.nonce).decode('utf-8'),
+        'tag': b64encode(tag).decode('utf-8'),
+        }
+    encryptedString = encryptedDict['cipher_text'] + '*' \
+        + encryptedDict['salt'] + '*' + encryptedDict['nonce'] + '*' \
+        + encryptedDict['tag']
+    return encryptedString
```
Comment (shravanasati) [inline encrypt.py:40]: why didnt we just use cryptocode lib instead of writing the whole encryption logic!?

**Review by shravanasati — COMMENTED** [review]
looks cool. but we need to do the database migration.

File: encrypt.py (hunk 2 above)
Comment (YashVardhan-AI) [reply encrypt.py:40]: oh just to flex and i did some smol changes

**Comment by YashVardhan-AI** [comment]
oh man now i will need to change all of it in momgo i think the login is ok in SQL

**Pull request merged**